src/config/config.ts

```diff
@@ -17,6 +17,7 @@ export default {
   API_JOURNEY_EVENT: "/journey",
   API_SESSION_INITIALISE: "/session/initialise",
   API_BUILD_PROVEN_USER_IDENTITY_DETAILS: "/user/proven-identity-details",
+  API_DCMAW_POLL: "/app/check-vc-receipt",
```

Comment:
Can we align this with the name of the function ie. `API_CHECK_MOBILE_APP_VC_RECEIPT`